packages/blockchain-wallet-v4-frontend/src/components/Flyout/Brokerage/Trade.tsx

```diff
@@ -159,7 +179,10 @@ const Trade = ({
         <FormattedMessage id='modals.trade.header' defaultMessage='Shortcuts' />
       </HeaderText>
       <IconWrapper onClick={handleClose}>
-        <Icon name='close' color='grey600' role='button' data-e2e='close' size='24px' cursor />
+        {/* @ts-ignore */}
+        <HoverIcon data-e2e='close' label='arrow up' size='md' color='grey600' role='button'>
```

Comment:
@pedroapfilho @awiermanncasas-bc 
1) should we have built in support for Icons to have a `cursor: pointer` attribute?  
2) typescript doesnt like the `role` attribute being added here. something we can fix?